p2p/tests/ser_deser.rs

```diff
@@ -81,6 +81,6 @@ fn test_capabilities() {
 fn test_pibd_capabilities() {
 	assert_eq!(
 		p2p::types::Capabilities::default() | p2p::types::Capabilities::PIBD_HIST,
-		p2p::types::Capabilities::from_bits_truncate(0b11111111 as u32),
+		p2p::types::Capabilities::from_bits_truncate(0b00011111 as u32),
```

Comment:
If I understand it correctly, the above value does an OR of default+PIBD bitmask. If I'm reading it correctly, the default already includes PIBD_HIST https://github.com/mimblewimble/grin/blob/010730e6daaa268a8316cc702c51fcb408fa0a6d/p2p/src/types.rs#L406
So maybe we can remove the `| p2p::types::Capabilities::PIBD_HIST` and correct the comment above the test which tells us that it is not in the defaults.

Reply:
Oh good catch. 👍 
I'll update in a separate PR (unrelated to this).

Reply:
https://github.com/mimblewimble/grin/pull/3565

Reply:
I'll clean this PR up once https://github.com/mimblewimble/grin/pull/3565 is merged.